intern/service/mesh/service.py

```diff
@@ -13,21 +13,23 @@
 # limitations under the License.
 
 from intern.service.service import Service
-from zmesh import Mesher
 from enum import IntEnum
 import numpy as np
 
+
 class VoxelUnits(IntEnum):
     """Enum with valid VoxelUnits
     """
-    nm = 1,
-    um = 1000,
-    mm = 1000000,
-    cm = 10000000,
-    nanometers = 1, 
-    micrometers = 1000, 
-    millimeters = 1000000, 
-    centimeters = 10000000,
+
+    nm = (1,)
```

Comment:
Why are these tuples, now?

Reply:
oh shoot i must have accidentally linted this full file. let me fix that...

Reply:
although — it looks like this comma-notation in the enum means that the interpreter was reading these types as `Tuple[int]`s anyway. The linter just added parentheses for clarity. I _think_ that's not what we wanted, but check this out:

```python
from enum import IntEnum

class X(IntEnum):
    a = (1,)
    b = (2,)

X.a == 1 # True
```

`IntEnum` does some weird stuff here (details on SO [here](https://stackoverflow.com/a/26125425/979255)); i have a feeling we've been using tuples all along...